paimon-format/src/main/java/org/apache/paimon/format/parquet/writer/RowDataParquetBuilder.java

```diff
@@ -45,15 +47,8 @@ public RowDataParquetBuilder(RowType rowType, Options conf) {
     public ParquetWriter<InternalRow> createWriter(OutputFile out, String compression)
             throws IOException {
 
-        String compressName = CompressionCodecName.SNAPPY.name();
-        if (null != compression) {
-            compressName = compression;
-        }
-
         return new ParquetRowDataBuilder(out, rowType)
-                .withCompressionCodec(
-                        CompressionCodecName.fromConf(
```

Comment:
In `org.apache.paimon.format.parquet.ParquetFileFormatFactory#supplyDefaultOptions` it will always set the `parquet.compression` to snappy. So the effective compression option is always snappy.